### PR TITLE
tweak idle banner

### PR DESCRIPTION
**Pull request by haraldschilly**
# Description
theme the idle banner, make the text below white and say the site name

# Testing Steps
1.
1.
1.

# Relevant Issues

### [Checklist](https://github.com/sagemathinc/cocalc/wiki/PR-Checklist):
- [ ] Run eslint on new and edited files
- [ ] All new code is actually used.
- [ ] Non-obvious code has some sort of comments.

Front end:
- [ ] Restart at least one project and check its `~/.smc/local_hub/local_hub.log`
- [ ] Completely restart Webpack with `./w` in `/src`
- [ ] Completely restart the hub by killing and restarting `./start_hub.py` in `/src/dev/project`
- [ ] Screenshots if relevant.

### CODE REVIEW

File: src/smc-webapp/client_browser.coffee [hunk 1]
```diff
@@ -43,10 +44,12 @@ misc = require('smc-util/misc')
 idle_notification_html = ->
     {redux}   = require('./app-framework')
     customize = redux.getStore('customize')
+    site_name = customize.get('site_name') or SITE_NAME
+    logo_url = customize.get('logo_square') or APP_LOGO_WHITE
     """
     <div>
-    <img src="#{APP_LOGO_WHITE}">
-    <h1>Collaborative Calculation</h1>
```
Comment (williamstein) [inline src/smc-webapp/client_browser.coffee:49]: In theme we have:

```
exports.SITE_NAME = "CoCalc";
exports.APP_TAGLINE = "Collaborative Calculation and Data Science";
```

So you're replacing "Collaborative Calculation" by CoCalc, and the result is going to be a big logo that says "CoCalc" with the words "CoCalc" right below it.  So... shouldn't the new heading be the tagline instead?

**Comment by williamstein** [comment]
See line comment.

File: src/smc-webapp/client_browser.coffee (hunk 1 above)
Comment (haraldschilly) [reply src/smc-webapp/client_browser.coffee:49]: ah, I see, the white logo also has the name. this must be done like on the about page.

Comment (williamstein) [reply src/smc-webapp/client_browser.coffee:49]: I don't know what you mean.  Incidentally, this file `src/smc-webapp/client_browser.coffee` is about to disappear as part of my client.coffee rewrite...

Comment (williamstein) [reply src/smc-webapp/client_browser.coffee:49]: So I'm going to merge this PR first as is, to avoid any merge conflict problems.

**Pull request merged**